webapp/backend/src/main/java/pl/splinter/backend/controller/ScanController.java

```diff
@@ -0,0 +1,23 @@
+package pl.splinter.backend.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import pl.splinter.backend.service.ScanData;
+import pl.splinter.backend.service.ScanService;
+
+@CrossOrigin("http://localhost:4200")
```

Comment:
@MateuszKepczynski can we create a separate file to store addresses? Or any other java solution for storing constant strings in one place. This address for sure will change when we dockerize or deploy it.